README.adoc

```diff
@@ -6,6 +6,37 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Operators
+
+In a new terminal, navigate to codeready-toolchain/host-operator directory and execute the following commands
+```
+$ minishift start --profile toolchain
+$ make up-local
+^C
+```
+In the same terminal, navigate to codeready-toolchain/member-operator directory and execute the following commands
+```
+$ make up-local
+^C
+$ make add-member-to-host
+```
+In the same terminal, navigate back to codeready-toolchain/host-operator directory and execute the following commands
+```
+make add-host-to-member
```

Comment:
same here: this goal does not exist

Reply:
@xcoulon right! it's sitting in the host/member repositories. So you'd have to switch to that repository to execute this. I had mentioned it above in the blurb before the code snippet - maybe there is another way to highlight it, so it is more clear. Easily gets lost among the rest of the words i think.